feature/platform/controllercard/tests/controller_card_redundancy_test/README.md

```diff
@@ -69,6 +69,11 @@
 * get last-reboot-time and compare with "previous-reboot-time"
   * "previous-reboot-time" must be smaller (earlier) then recently collected last-reboot-time
 
+## Canonical OC
+```json
+{}
+```
+
 ## Config Parameter coverage
 
 *   /components/component/controller_card/config/power-admin-state
```

Comment:
This path should be moved to the heading `## OpenConfig Path and RPC Coverage` in yaml format.